### ISSUE TITLE
Fortoresse XY

### ISSUE THREAD

**Issue by hydroper** [fewfre/FortoresseDressroom#2]
Hey, @fewfre, if you want to give a look

https://github.com/hydroper/fortoresseXY

I'm using a different architecture from the one Atelier 801 used in their multiplayer games. Mine is based on Valve Source, which sends high frequency packets from a server to clients (based on my previous experience trying a different game).

Note that the game's title is inspired by the Pokémon XY title.

**Comment by fewfre** [comment]
Hey, glad someone else remembers Fortoresse's glory days!

I wish you the best of luck, and look forward to an alpha build!

**Comment by hydroper** [comment]
Glad you liked the project!

I've made pretty much structure, but I'm wondering, what is your opinion on discarding TypeScript at the server side?

I've combined TypeScript + Node.js + Prisma ORM + Express.js and got a little TypeScript version conflict when installing the `cors` package (requires 3.x-4.0).

I've not a good view of TypeScript and have discarded it from my codebase, but in turn writing JSDoc comments seems to be a lot more verbose and time consuming! I'm really worried now...
